frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -223,9 +256,19 @@ function MainMenu(props: Props): ReactElement {
       ...S4AMenuOptions,
     ]
   } else {
```

Comment:
Yes. Some of these ideas are good ones that I wanted to do. Unfortunately, the move to BaseWeb made these hard.

Thought 1 - I preferred the idea of creating a separate divider. In fact, that was there before I made the changes to Dropdown. Unfortunately, BaseWeb was more prescriptive that the items provided needed to be actual options. The problem then became that the divider became an option that BaseWeb thought was selectable. I tried doing some work to inform that the menu option is not selectable, but BaseWeb still considered it an option in their list. This broke some A11y issues and caused some weird UX. For example, Using the keyboard to navigate options made the divider still an option. I could create a divider option, and then create a function that applies those changes in. It could also flatten it and do what Thought 2 is. I'll investigate how hard that is.

Reply:
Added an updated look that I think looks better. Let me know if you think the same.